web/src/themes/lotus.css

```
@@ -0,0 +1,108 @@
+  --chart-3: oklch(0.5421 0.0702 102.4215);
+  --chart-4: oklch(0.5707 0.1715 16.4775);
+  --chart-5: oklch(0.5846 0.1185 359.7673);
+  --sidebar: oklch(0.8927 0.0596 99.0700);
+  --sidebar-foreground: oklch(0.4519 0.0259 285.3160);
+  --sidebar-primary: oklch(0.5211 0.0862 261.4868);
+  --sidebar-primary-foreground: oklch(0.9360 0.0625 101.5859);
+  --sidebar-accent: oklch(0.5211 0.0862 261.4868);
+  --sidebar-accent-foreground: oklch(0.9360 0.0625 101.5859);
+  --sidebar-border: oklch(0.8462 0.0576 99.7960);
+  --sidebar-ring: oklch(0.5211 0.0862 261.4868);
+  --font-sans: Inter;
```

Stylelint (17.13.0)

[error] Expected "Inter" to be "inter" (value-keyword-case)

(value-keyword-case)